AABB_tree/include/CGAL/AABB_polyhedron_triangle_primitive.h

```diff
@@ -154,8 +155,8 @@ namespace CGAL {
     };  // end class AABB_polyhedron_triangle_primitive
 
 
-
 }  // end namespace CGAL
+/// \endcond
```

Comment:
I disagree with that choice. For our users, it is a lot more useful to get a documentation of the class, with the deprecation notice, mentioning by what it is should replaced, instead of having the documentation missing.